tests/webview/ui/test_content.py

```diff
@@ -766,3 +766,18 @@ def test_chapter_review_version_matches_book_version(webview_base_url, selenium,
 
     # THEN the chapter review version matches the book version
     assert content.page_version == content.book_version
+
+
+@markers.webview
+@markers.test_case('C195071')
+@markers.nondestructive
+@markers.parametrize('id', ['yWshIYVW@1'])
+def test_buggy_msg_not_show_in_left_panel(webview_base_url, id, selenium):
+    # GIVEN the webview base url, a chapter page id, and the Selenium driver
+
+    # WHEN we visit that page of the chapter
+    content = Content(selenium, webview_base_url, id=id).open()
+
+    # THEN correct book info is listed in the left panel
```

Comment:
You have the idea down of using the POM and executing methods for the test but what happens here if `is_left_panel_title_displayed` is false?

The answer is the test passes. Is that what we want?